### PR TITLE
Reference fix

### PR DESCRIPTION
**Pull request by cloveras**
This needs fixing: https://github.com/vippsas/vipps-report-api/blob/reference-fix/report-api-faq.md#how-can-i-get-the-details-for-a-payment-i-see-in-my-bank

Slack: https://vippsas.slack.com/archives/C044EBKCS3D/p1700744429497149

### CODE REVIEW

File: report-api-faq.md [hunk 1]
```diff
@@ -163,7 +163,10 @@ The payout details can be found with these endpoints, where `{topic}` is `funds`
 * [`GET:/report/v2/ledgers/{ledgerId}/{topic}/dates/{ledgerDate}`][fetch-report-by-date-endpoint]
 * [`GET:/report/v2/ledgers/{ledgerId}/{topic}/feed`][fetch-report-by-feed-endpoint]
 
-Match the settlement ID in the bank with `pspReference`.
+Match the `reference` in the bank statement with the `pspReference` in the Report API.
```
Comment (dss-vipps) [inline report-api-faq.md:166]:
```suggestion
Match the reference on the transfer from your bank statement with the `reference` in the Report API.
```

File: report-api-faq.md [hunk 2]
```diff
@@ -163,7 +163,10 @@ The payout details can be found with these endpoints, where `{topic}` is `funds`
...
+When you retrieve a series of payments with
+[`GET:/report/v2/ledgers/{ledgerId}/{topic}/feed`](https://developer.vippsmobilepay.com/api/report/#tag/reportv2ledgers/paths/~1report~1v2~1ledgers~1%7BledgerId%7D~1%7Btopic%7D~1feed/get)
+in the Report API, each payment’s `pspReference` is the `reference` for the bank payout.
```
Comment (dss-vipps) [inline report-api-faq.md:169]: No I don't understand this one...   what is a "payment" here?

Comment (rebekaburnett) [reply report-api-faq.md:169]: @dss-vipps Hi! Does he mean "Payment entries" or "payment reports" ? something like that? If you think of a better way, just go ahead and commit it and them merge this thing so it doesn't get too old while he's away. I'm going on vacation now, too. TaTa! :-)

Comment (cloveras) [reply report-api-faq.md]: Tried to make it clearer. Did I succeed..?

File: report-api-faq.md [hunk 3]
```diff
@@ -107,13 +107,21 @@ available in the settlement reports on
 or on
 [SFTP report (deprecated)](https://developer.vippsmobilepay.com/docs/settlements/sftp-report-service).
 
+Match the reference on the transfer from your bank statement with the `reference` in the Report API.
+When you retrieve a series of payments (made by users in the app) with
+[`GET:/report/v2/ledgers/{ledgerId}/{topic}/feed`](https://developer.vippsmobilepay.com/api/report/#tag/reportv2ledgers/paths/~1report~1v2~1ledgers~1%7BledgerId%7D~1%7Btopic%7D~1feed/get)
+in the Report API, each of those payments' `pspReference` is the `reference` for the bank payout.
```
Comment (dss-vipps) [inline report-api-faq.md:113]: Don't have much time now, but this is wrong..

**Review by dss-vipps — CHANGES_REQUESTED** [review]

File: report-api-faq.md (hunk 3 above)
Comment (cloveras) [reply report-api-faq.md:113]: Yes, I had a conflict and it looks like I mangled it. Will make a new commit. Thanks so far!

Comment (cloveras) [reply report-api-faq.md:113]: Changed, hopefully correct now. 🤞

**Comment by rebekaburnett** [comment]
I don't seen any changes here 🤷‍♀️

**Review by rebekaburnett — APPROVED** [review]

**Pull request merged**